src/components/tile/tile.stories.tsx

```diff
@@ -582,3 +589,191 @@ export const WithAccordionAndTileFooter: ComponentStory<typeof Tile> = () => (
 WithAccordionAndTileFooter.parameters = {
   chromatic: { disableSnapshot: true },
 };
+
+export const ResponsiveDefaultStory: ComponentStory<typeof Tile> = () => (
```

Comment:
Suggestion: These are great for illustrating these enhancements to Carbon consumers. I wonder if it might be worth adding similar ones to the `test` stories with some viewport adjustments so chromatic can capture when the containers wraps and dividers etc are shown. We don't need to worry about them being public facing here so we can just focus on making sure the we have any future visual regressions captured. If you're happy to do that I would disable chromatic for these ones and let it take baselines from the test ones

```
StoryName.parameters = {
  chromatic: {
    viewports: [1500, 900],
  },
};

Reply:
This is non-blocking for me, if it's going to cause issues with you're deadline we can make these changes in a new PR